UnitsNet.Tests/CustomCode/EnergyTests.cs

```diff
@@ -63,6 +58,17 @@ public class EnergyTests : EnergyTestsBase
 
         protected override double WattHoursInOneJoule => 0.000277777778;
 
+        protected override double ElectronVoltsInOneJoule => 6.2415093433e+18;
+
+        protected override double KiloelectronVoltsInOneJoule => ElectronVoltsInOneJoule / 1_000;
```

Comment:
Sorry, I was maybe not clear, these values still need to be constants and not computations. The idea is to double check the computations in the conversion functions, preferably from a reference like a website or conversion calculator.

Please change these back to how they were, but use the same constant value as in electron volts and use scientific notation: `KiloelectronVoltsInOneJoule => 6.2415093433e+15` etc.

Reply:
Here are the conventions we use:
https://github.com/angularsen/UnitsNet/wiki/Adding-a-New-Unit#conventions